internal/target/target.go

```diff
@@ -71,6 +71,8 @@ func UnmarshalTargetOptions(targetName string, rawOptions json.RawMessage) (Targ
 		options = new(AWSTargetOptions)
 	case "org.osbuild.gcp":
 		options = new(GCPTargetOptions)
+	case "org.osbuild.azure.image":
```

Comment:
Wouldn't it make sense to have this case right under `org.osbuild.azure`? Not really a big issue...

Reply:
Good point, will move it in some kind of a follow-up.